### ISSUE TITLE
#941 broke tfjs in @observablehq

### ISSUE THREAD

**Issue by Fil** [tensorflow/tfjs#960]
https://github.com/tensorflow/tfjs/pull/941 broke tfjs in @observablehq:

before:
`require("@tensorflow/tfjs@0.13.5")`
results in a `tf` object.

after:
`require("@tensorflow/tfjs@0.14.0")`
results in an **"Error: invalid module"** message

See https://beta.observablehq.com/d/4010e96b9684112a

**Comment by tafsiri** [comment]
I suspect it stems from `require('node-fetch')` which results in the same message if one tries to load it directly. It ultimately resolves to https://unpkg.com/node-fetch@2.3.0/browser.js and errors out on the `module.exports` line (saying that module is not defined).

**Comment by kangyizhang** [comment]
Yes I suspect it's because of rollup configuration of tfjs-data. However I can not reproduce this issue on my local machine. Do you know how I can reproduce this issue with local bundle .min.js file or tarball package?

**Comment by Fil** [comment]
You can try `require("http://localhost/path/to/tf.min.js")` ; if your browser doesn't let you http on localhost or 127.0.0.1, then you'll need to use an https server (I do that with neocities usually ^^).

**Comment by mbostock** [comment]
I have proposed a fix in tensorflow/tfjs-data#102, but I haven’t rebuilt the top-level tfjs to test.

You can workaround this issue in the meantime by aliasing node-fetch:

```js
tfjs = require.alias({"node-fetch": null})("@tensorflow/tfjs@0.14")
```
